community/hacktoberfest/index.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: single
+title:  "Hacktoberfest"
+sidebar:
+  nav: community
+---
+
+[Hacktoberfest](https://hacktoberfest.digitalocean.com/) is a month-long global celebration of open source software that happens every year in October. During this event, anyone can support open source by contributing changes, which earn swag and recognition from the [Continuous Delivery Foundation](https://cd.foundation), our parent organization.
+
+We invite everyone to participate and to contribute to the Spinnaker project, regardless of their background and Spinnaker experience. There are many ways to contribute to Spinnaker during Hacktoberfest and other contributor events. You can work on code or documentation, assist community members in [Spinnaker Slack](http://join.spinnaker.io), or answer [StackOverflow questions](https://stackoverflow.com/questions/tagged/spinnaker). You can also share your experiences with Spinnaker by creating a new blog post, video, or social media steam. For more information on eligible contributions, see the [What to Hack](/community/gardening/what-to-hack/) page. For help getting started as a Spinnaker contributor, visit the [Contributing](/community/contributing/) page.
```

Comment:
I'm in favor of breaking out items into a list: 
* work on code or docs
* assist community members
* answer StackOverflow questions

Reply:
Like this idea, doing it. Otherwise ppl won't read it, good call